app/Http/Resources/ResourceResource.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class ResourceResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @return array<string, mixed>
+     */
+    public function toArray(Request $request): array
+    {
+        return ['id' => $this->id,
+            'name' => $this->name,
+            'county' => IdAndNameResource::make($this->county),
+            'organization' => IdAndNameResource::make($this->organisation),
+            'category' => IdAndNameResource::make($this->category),
+            'subcategory' => IdAndNameResource::make($this->subcategory),
+//            'types' => IdAndNameResource::collection($this->types),
```

Comment:
```suggestion
            'types' => IdAndNameResource::collection($this->types),
```

de ce e comentată linia asta? ai dat de vreo problemă?